lti_consumer/tests/unit/test_lti.py

```diff
@@ -199,6 +188,20 @@ def test_get_signed_lti_parameters(self):
         del expected_lti_parameters['launch_presentation_locale']
         self.assertEqual(self.lti_consumer.get_signed_lti_parameters(), expected_lti_parameters)
 
+    @patch_signed_parameters
+    def test_parameter_processors(self):
+        self._update_xblock_for_signed_parameters()
+        self.xblock.enable_processors = True
+
+        with patch('lti_consumer.lti_consumer.LtiConsumerXBlock.settings', PropertyMock(return_value={
+            'parameters_processors': [
+                'lti_consumer.tests.unit.test_utils:foo_processor',
+            ],
+        })):
+            params = self.lti_consumer.get_signed_lti_parameters()
+            assert '' == params['custom_author_country']
+            assert 'author@example.com' == params['custom_author_email']
```

Comment:
Please use the standard `self.assertEqual()` and other assertion methods from `unittest.TestCase`.

Reply:
Great catch. Thanks @ormsbee.

To my best understanding we're moving away from assertion methods to native `assert` statements. Could you please let me know if that's correct?

For consistency it makes sense, but as far as I can tell, `pytest` with native `assert` is far superior to what `unittest.TestCase` can offer in terms of debugging capabilities.

Reply:
Interesting. I didn't realize that pytest recommended native `assert` or supported it so well. I'll read up on it today.

Reply:
Thanks @ormsbee!